cli/factory.go

```diff
@@ -79,8 +79,8 @@ func getUnixListener(path string) (net.Listener, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err = unix.Unlink(path); err != nil && !os.IsNotExist(err) {
-		return nil, err
+	if _, err = os.Stat(path); err == nil || os.IsExist(err) {
```

Comment:
There is no `os.IsExist` error for `os.Stat`.

Reply:
> There is no `os.IsExist` error for `os.Stat`.

Submit a new version according to your comments.